en/services/gimbal_v2.md

```diff
@@ -235,14 +230,12 @@ The autopilot will have to act as the *Gimbal Manager* and provide the driver an
 
 #### What about RC (non-MAVLink) control?
 
-The autopilot needs to be configured to either accept MAVLink input (so [GIMBAL_MANAGER_SET_ATTITUDE](#GIMBAL_MANAGER_SET_ATTITUDE)) or RC control.
+The autopilot needs to be configured to either accept MAVLink input (so [GIMBAL_MANAGER_SET_MANUAL_CONTROL](#GIMBAL_MANAGER_SET_MANUAL_CONTROL)) or RC control.
+In both cases, the message is then to be forwarded to the gimbal device using [GIMBAL_DEVICE_SET_MANUAL_CONTROL](#GIMBAL_DEVICE_SET_MANUAL_CONTROL).
 
 For RC control, the channels will have to be manually mapped/configured to control the gimbal.
-It is up to the gimbal manager implementation to deconflict between RC and MAVLink input.
-This is in the same way that also RC input to fly needs to be selected from either RC or MAVLink and is up to the implementation.
-The recommendation is to make it configurable using for instance a parameter.
-
-
+This is the same approach as is used for managing the input source for flying; it is up to the implementation to select either RC or MAVLink.
```

Comment:
Please check this sentence; it is my interpretation of you sentence, but I might have it wrong.

Reply:
I think that's right.